### ISSUE TITLE
Proxy authentication broken in 1.45 when installing extensions

### ISSUE THREAD

**Issue by ericjmcd** [microsoft/vscode#97757]
- VSCode Version:
- OS Version:

Steps to Reproduce:

1. Attempt to install an extension behind a proxy requiring authentication.
2. Enter username/password on popup - clicking OK or Cancel doesn't do anything. Only way to close the pop-up is to click X which gives a 2nd popup (same behavior). Then the install fails.

Note that the March release 1.44 works fine.

Does this issue occur when all extensions are disabled?: N/A

**Comment by dliberat** [comment]
As a followup, this issue _does_ occur when all extensions are disabled. I get the following log output when I launch the program via the command line with `--verbose`.

```
[main 2020-05-14T00:36:18.272Z] RequestService#request https://update.code.visualstudio.com/api/update/win32-x64-user/stable/d69a79b73808559a91206d73d7717ff5f798f23c
[main 2020-05-14T00:36:18.316Z] Shared process: init ready
[main 2020-05-14T00:36:20.287Z] [uncaught exception in main]: Error: Script failed to execute, this normally means an error was thrown. Check the renderer console for the error.
[main 2020-05-14T00:36:20.287Z] Error: Script failed to execute, this normally means an error was thrown. Check the renderer console for the error.
    at WebFrame../lib/renderer/api/web-frame.ts.WebFrame.<computed> [as executeJavaScript] (electron/js2c/renderer_init.js:3005:33)
    at electron/js2c/renderer_init.js:4278:43
    at electron/js2c/renderer_init.js:3907:40
    at new Promise (<anonymous>)
    at EventEmitter.<anonymous> (electron/js2c/renderer_init.js:3907:9)
    at EventEmitter.emit (events.js:203:13)
    at Object.onMessage (electron/js2c/renderer_init.js:3675:16)
[main 2020-05-14T00:36:20.385Z] [uncaught exception in main]: Error: Script failed to execute, this normally means an error was thrown. Check the renderer console for the error.
[main 2020-05-14T00:36:20.386Z] Error: Script failed to execute, this normally means an error was thrown. Check the renderer console for the error.
    at WebFrame../lib/renderer/api/web-frame.ts.WebFrame.<computed> [as executeJavaScript] (electron/js2c/renderer_init.js:3005:33)
    at electron/js2c/renderer_init.js:4278:43
    at electron/js2c/renderer_init.js:3907:40
    at new Promise (<anonymous>)
    at EventEmitter.<anonymous> (electron/js2c/renderer_init.js:3907:9)
    at EventEmitter.emit (events.js:203:13)
    at Object.onMessage (electron/js2c/renderer_init.js:3675:16)
[main 2020-05-14T00:36:43.942Z] update#checkForUpdates, state =  idle
[main 2020-05-14T00:36:43.942Z] update#setState checking for updates
```

**Version Info**
Version: 1.45.0 (user setup)
Commit: d69a79b73808559a91206d73d7717ff5f798f23c
Date: 2020-05-07T16:18:48.860Z
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Comment by likejie9502** [comment]
ERR Script failed to execute, this normally means an error was thrown. Check the renderer console for the error.: Error: Script failed to execute, this normally means an error was thrown. Check the renderer console for the error.
    at WebFrame../lib/renderer/api/web-frame.ts.WebFrame.<computed> [as executeJavaScript] (electron/js2c/renderer_init.js:3005:33)
    at electron/js2c/renderer_init.js:4278:43
    at electron/js2c/renderer_init.js:3907:40
    at new Promise (<anonymous>)
    at EventEmitter.<anonymous> (electron/js2c/renderer_init.js:3907:9)
    at EventEmitter.emit (events.js:203:13)
    at Object.onMessage (electron/js2c/renderer_init.js:3675:16)

**Comment by deepak1556** [comment]
The fix for this has already been made in master, can you try our insiders build to confirm https://code.visualstudio.com/insiders/

**Comment by chrmarti** [comment]
/duplicate https://github.com/microsoft/vscode/issues/97199

Fix will be in 1.45.1.
